README.md

```diff
@@ -1,4 +1,105 @@
+
+* `asset`: A DID identifying the asset to which permissions are being granted, such as a specific NFT (e.g., did:nft:137:0x3330000000000000000000000000000000000000_31415).
+* `permissions`: An array detailing the specific permissions being granted. Each permission typically includes a name (e.g., "commands", "location:View:Current") and a human-readable description.
+* `attachments`: An array of documents related to the permission agreement, each with a name, description, contentType, and uri (e.g., for a legal agreement).
+* `signatures`: An array of signatures from the involved parties, including the signer's address, the signature itself, and a timestamp.
+
+### Use Case: Payments
+
+When the type within the agreement array is set to "payment", the structure defines the terms of a financial transaction. Relevant fields within this agreement type (as seen in [`sacd.payment.example.json`](sata/sacd.payment.example.json)) include:
+
+* `asset`: A DID identifying the payment asset, such as a specific ERC-20 token (e.g., did:erc20:137:0x4440000000000000000000000000000000000000).
+* `payment`: An object containing the payment details:
+    * `amount`: The amount to be paid (e.g., "125000000000000000000000").
+    * `recurrence`: The frequency of payment (e.g., "monthly", "one-time").
+    * `terms`: Additional payment terms, which can include:
+        * `initialPayment`: The initial payment amount.
+        * `paymentMethod`: The method of payment (e.g., "direct transfer").
+* `purpose`: A description of what the payment is for (e.g., "Subscription service for vehicle data").
+* `attachments`: An array of documents related to the payment agreement, each with a name, description, contentType, and uri (e.g., for a service agreement or payment schedule).
+* `signatures`: An array of signatures from the involved parties, including the signer's address, the signature itself, and a timestamp.
+
+This documentation should provide a foundational understanding of the SACD JSON format and its usage for defining payment and permission agreements within the DIMO ecosystem. More complex agreements may include additional fields within the agreement section or utilize the extensions field for specific requirements.
+
+### NFT DID Format
+
+The DID (Decentralized Identifier) follows the format:
+```
+did:nft:<chainId>:<contractAddress>_<tokenId>
+```
+
+Example: did:nft:137:0xbA5738a18d83D41847dfFbDC6101d37C69c9B0cF_3
+
+Where:
+
+* `did:nft`: is the prefix that identifies the asset type (e.g. nft, erc20, fiat)
+* `<chainId>` is the numeric ID of the blockchain (e.g., 137 for Polygon)
+* `<contractAddress>` is the 0x address of the asset
+* `<tokenId>` is the numeric ID of the specific token
```

Comment:
Is this optional? Probably want to spec it to optional if not all DIDs will include `tokenId`

Reply:
The section is called "NFT DID Format" so I did not complain about this.